js/components/Debugger.tsx

```diff
@@ -57,10 +66,38 @@ export const Debugger = ({ apple2 }: DebuggerProps) => {
         debug?.step();
     }, [debug]);
 
-    const doMemoryPage = useCallback((event: JSX.TargetedMouseEvent<HTMLInputElement>) => {
+    const doLoadAddress = useCallback((event: JSX.TargetedEvent<HTMLInputElement>) => {
+        setLoadAddress(event.currentTarget.value);
+    }, []);
+    const doRunCheck = useCallback((event: JSX.TargetedEvent<HTMLInputElement>) => {
+        setRun(event.currentTarget.checked);
+    }, []);
+
+    const doMemoryPage = useCallback((event: JSX.TargetedEvent<HTMLInputElement>) => {
         setMemoryPage(event.currentTarget.value);
     }, []);
 
+    const doChooseFile = useCallback((handles: FileSystemFileHandle[]) => {
+        if (debug && handles.length === 1) {
+            spawn(async () => {
+                const file = await handles[0].getFile();
+                let atAddress = parseInt(loadAddress, 16) || 0x800;
+
+                const matches = file.name.match(CIDERPRESS_EXTENSION);
+                if (matches && matches.length === 3) {
+                    const [, , aux] = matches;
+                    atAddress = parseInt(aux, 16);
+                }
```

Comment:
This means there's no way to override the address in the CiderPress extension, right?